aqua/djview/Portfile

```diff
@@ -21,33 +21,42 @@ homepage            http://djvu.sourceforge.net/djview4.html
 
 master_sites        sourceforge:project/djvu/DjView/4.12/
 
-patchfiles          patch-powerpc.diff 
-                    #patch-makefile-qmake-workaround.diff 
+patchfiles          patch-powerpc.diff
 
-checksums           rmd160  92108c2c92b2593cb85dc288a578ed7440671838 \
-                    sha256  5673c6a8b7e195b91a1720b24091915b8145de34879db1158bc936b100eaf3e3 \
-                    size    713842
+checksums           rmd160  7190a27b20118b877c2d58a6a94afbf8319dc513 \
+                    sha256  17bfb9731ab8070e01235381f08e57b1231ffb5f49eb031873ebfa189cdaf47d \
+                    size    1986642
 
-depends_lib-append \
-    port:djvulibre
+depends_lib-append  port:djvulibre
+depends_build-append \
+                    path:bin/pkg-config:pkgconfig
 
-worksrcdir          djview4-${version}
-
-universal_variant yes
+universal_variant   yes
 use_autoreconf      yes
 
-if {$subport eq $name} {
+if {${os.major} < 17} {
```

Comment:
It is probably worth to add `${os.platform} eq "darwin"` to ensure Qt4 is not picked on Linux (at least here in upstream it is not supported on Linux at all).